vue/src/router/index.js

```diff
@@ -1,21 +1,48 @@
 import Vue from 'vue';
-import VueRouter from 'vue-router';
-import Home from '../views/Home.vue';
-Vue.use(VueRouter);
+import Router from 'vue-router';
 
-const routes = [{
-  path: '/',
-  name: 'noticeBoard',
-  // route level code-splitting
-  // this generates a separate chunk (about.[hash].js) for this route
-  // which is lazy-loaded when the route is visited.
-  component: () =>
-    import ('../views/NoticeBoard.vue')
-}
-];
+import RouterView from '@/views/RouterView.vue';
+import { i18n } from '@/main.js';
 
-const router = new VueRouter({
-  routes
-});
+Vue.use(Router);
+
+export default new Router({
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes: [{
+    //check for language prefix (i.e /en/ or /de/). Default to /en/ if no valid prefix was found
```

Comment:
No language prefixes needed
We use local variable (https://github.com/kazupon/vue-i18n/issues/2)